interfaces/cython/cantera/test/test_convert.py

```diff
@@ -459,3 +459,34 @@ def test_reaction_orders(self):
         self.assertNear(R.orders.get('OH'), 0.15)
         self.assertTrue(R.allow_negative_orders)
         self.assertNear(R.orders.get('H2'), -0.25)
+
+    #Here we are testing if passing a very long string will result in a Solution
```

Comment:
I'd actually prefer if these were a docstring of the function, instead of a comment above the function. Comments like this have a bad habit of being left behind if the function is moved.